.github/workflows/maven.yml

```diff
@@ -45,11 +45,12 @@ jobs:
       with:
         tool-cache: false
         large-packages: false
-    - name: Checkout ci.common
+    - name: Checkout dshimo ci.common
       uses: actions/checkout@v3
       with:
-        repository: OpenLiberty/ci.common
+        repository: dshimo/ci.common
```

Comment:
You need to change this in the Windows section also. That means the windows builds/tests did not use your branch.

Reply:
For Windows, change the `OpenLiberty` to your gh id and add `--branch varProcessing --single-branch` to the `git clone`.